src/generateZeroFunction.js

```diff
@@ -12,7 +12,9 @@
  *
  */
 function generateZeroFunction() {
-  // write code here
+  return function(zero) {
```

Comment:
в нашем случае функция ничего не принимает
```suggestion
  return function() {
```